Search API — Add `facetStats` description (#224)
meilisearch/specifications: text/0118-search-api.md

If a field distributed as a facet contains no value, it is returned as a `facetDistribution` field with an empty object as value.

> See [3.1.4. `facets`](#314-facets) section.

#### 3.2.10. `facetStats`

- Type: Object
- Required: False

> When using the `facets` parameter, the distributed facets that stricly contain numeric values are displayed in a `facetStats` object that contains, per facet, the numeric `min` and `max` values of the hits returned by the search query.

Reviewer: We don't ignore a facet as soon as one of the hit contains a non-numeric value, but if all of the hits contain non-numeric values.

```suggestion
When using the `facets` parameter, the distributed facets that contain some numeric values are displayed in a `facetStats` object that contains, per facet, the numeric `min` and `max` values of the hits returned by the search query.

If none of the hits returned by the search query have a numeric value for a facet, this facet is not part of the `facetStats` object.
```

Author: Thanks, @dureuill!

If a field distributed as a facet contains no value, it is returned as a `facetDistribution` field with an empty object as value.

> See [3.1.4. `facets`](#314-facets) section.

#### 3.2.10. `facetStats`

- Type: Object
- Required: False

> When using the `facets` parameter, the distributed facets that contain some numeric values are displayed in a `facetStats` object that contains, per facet, the numeric `min` and `max` values of the hits returned by the search query.

Reviewer 2: Do they represent the min and max value of the `hits` in `{ hits: [] }` or the min-max value of the remaining candidates?

Reviewer: B

Reviewer 2: In which case, I don't  this sentence means `B`: 

> in a `facetStats` object that contains, per facet, the numeric `min` and `max` values of the hits returned by the search query.

Reviewer: In that case, suggesting:

```suggestion
When using the `facets` parameter, the distributed facets that contain some numeric values are displayed in a `facetStats` object that contains, per facet, the numeric `min` and `max` values for that facet of all candidate documents matching the search query.
```

I'm not sure about the "remaining candidates" phrasing, as we return the min-max of *all* candidates (and same for facet distribution?)

Reviewer 2: I like your suggestion!
